assets/messages.yml

```
@@ -447,6 +447,31 @@ messages:
 
         If you haven't already, you might like to make use of the [*+search feature+*|https://bugs.mojang.com/issues/?jql=project=%project_id%] to see if the issue has already been mentioned.
 
+        %quick_links%
+      fillname:
+        - Enter parent issue ID
+  duplicate-wf:
+    - project:
+        - bds
+        - mc
+        - mcapi
+        - mcd
+        - mce
+        - mcl
+        - mcpe
+        - mctest
+        - realms
+        - web
+      name: Duplicate (wf)
```

```suggestion
      name: Duplicate (won't fix)
```
Maybe write out "Won't Fix" here so it is more clear